esmvaltool/recipes/ref/recipe_model_benchmarking_boxplots_region_trend.yml

```diff
@@ -0,0 +1,177 @@
+
+  trend_psl:
+    custom_order: true
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+    detrend:
+      method: constant
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: m-1 kg s-2 year-1
+
+  trend_land:
+    custom_order: true
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    mask_landsea:
+      mask_out: sea
+    area_statistics:
+      operator: mean
+    detrend:
+      method: constant
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: K year-1
+
+  trend_pr:
+    custom_order: true
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+    detrend:
+      method: constant
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: mm kg m-3 day-1 year-1
+
+  trend_sst:
+    custom_order: true
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    mask_below_threshold:
+      threshold: 273.15
+    area_statistics:
+      operator: mean
+    detrend:
+      method: constant
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: K year-1
+
+diagnostics:
+  plot_boxplots_trend:
+    description: Plot boxplots for different variables.
+    variables:
+      tas_land:
+        <<: *var_settings
+        preprocessor: trend_land
+        short_name: tas
+        additional_datasets:
+          - {dataset: HadCRUT5, project: OBS, type: ground,
+             version: 5.0.1.0-analysis, tier: 2}
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3}
+      pr:
+        <<: *var_settings
+        preprocessor: trend_pr
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, version: 2.3,
+             tier: 2}
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3}
+      sst:
+        <<: *var_settings
+        preprocessor: trend_sst
+        short_name: ts
+        additional_datasets:
+          - {dataset: HadISST, project: OBS, type: reanaly, version: 1, tier: 2}
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3}
+    scripts:
+      allplots:
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        group_variables_by: variable_group
+        plots:
+          benchmarking_boxplot:
+            var_order: ['tas_land', 'sst', 'pr']
+        figure_kwargs:
+          figsize: [4, 5]
+          layout: constrained
+        # Overwrites metric in title, necessary, if no distance metric is used
+        title_metric: 'Linear trend (2001-2014)'
```

Comment:
```suggestion
        pyplot_kwargs:
          suptitle: Linear of trend of MIROC for N.Europe
```

Reply:
Does not effect the suptitle, see above?